### DISCUSSION TITLE
Next.js Live - TypeError: Cannot convert undefined or null to object

### DISCUSSION THREAD

**Discussion by hakudev** [vercel/community#360] [Help] [answered]
I've deployed an example project on Vercel with the **Live** feature and ran into an error when trying to access the collaboration view.
The testing repositories were created on BitBucket. Both the nextjs-docs and nextjs-commerce example show this error. 

![image](https://user-images.githubusercontent.com/6309576/154070825-51838ce5-4ee4-4f55-b00a-02d7b2dd4664.png)

I'm assuming there's an error utilizing memfs:
![image](https://user-images.githubusercontent.com/6309576/154072905-9af111b5-4567-45e1-acbc-25aa5623b708.png)

This problem occurs if the repository is created on BitBucket instead of GitHub. I've used a team account with a pro trial.

**Any solutions yet?**

**Comment by becca** [comment, marked as answer]
👋 hey there @hakudev. Next.js Live is still in a private beta and at the moment we only support the GitHub integration/repositories. 

I will work with the team to ensure we have clearer messaging in place for this error (thank you for the screenshot!) and make it clear on vercel.com/live that GitHub repositories should be used right now.
